test/powershell/Modules/Microsoft.WSMan.Management/ConfigProvider.Tests.ps1

```diff
@@ -0,0 +1,478 @@
+Describe "WSMan Config Provider" -Tag Feature,RequireAdminOnWindows {
+    BeforeAll {
+        #skip all tests on non-windows platform
+        $originalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        $PSDefaultParameterValues["it:skip"] = !$IsWindows
+
+        if ($IsWindows) {
+            $badCredentialError = 1326
+            $pluginXml = [xml](winrm g winrm/config/plugin?name=microsoft.powershell -format:xml)
```

Comment:
I'll make the change

Reply:
Had to change it back as using Install-PowerShellRemoting or Enable-PSRemoting was making things complicated even though my tests had no dependency on remoting actually working.  Since I'm just relying on a well known existing plugin configuration, I think using microsoft.powershell seems reasonable.